src/web/app/pages-instructor/instructor-course-enroll-page/instructor-course-enroll-page.component.html

```diff
@@ -49,6 +49,14 @@ <h1>Enroll Students for {{ courseId }}</h1>
                   [hidden]="isExistingStudentsPanelCollapsed" @collapseAnim>
               </hot-table>
             </div>
+
+            <div *ngIf="showEnrollErrorMessage && enrollErrorMessage" class="card card-default">
```

Comment:
Is there a case whereby `enrollErrorMessage` is non-empty but `showEnrollErrorMessage` is `false`? If not, the condition `showEnrollErrorMessage` is actually redundant.